docs/content/config.md

```diff
@@ -77,6 +77,13 @@ resolver:
 # Optional: set to skip running `go mod tidy` when generating server code
 # skip_mod_tidy: true
 
+# Optional: set to modify the initialisms regarded for Go names
+#go_initialisms:
+#  replace_defaults: true # if true, the default initialisms will get dropped in favor of the new ones instead of being added
```

Comment:
Fixed it. That said, I noticed that at some places you use more indentation for list items than at other places (for example `config.yml` line `31` vs `config.md` line `100`. Which way do you prefer?

Reply:
Ugh. Yeah. The `config.md` is especially hard because it's got a markdown code block with commented out yaml, AND for list items yaml tolerates the removal of a level of indentation that we would naturally expect to occur.
See https://stackoverflow.com/a/71968274/2904179

Reply:
Done! I've also adjusted the `config.yml`.

Reply:
Thanks!